corya-akamai/manager
# feat: [UIE-8054] - DBaaS enhancements 1

Description 📝
Common changes for the V2 enhancements that will be used in create/landing/detail flows

Changes 🔄
List any change relevant to the reviewer.
- Adding capability to accounts and regions
- Check for either capability to show menu item
- Use V2 to add Beta flag
- Add additional factory data to test flows

Target release date 🗓️
9/3/24

How to test 🧪
Prerequisites
(How to setup test environment)

Add V2 capability to account factory mocks
Verification steps
(How to verify changes)

Beta chip shows for V2 capability
As an Author I have considered 🤔
Check all that apply

 👀 Doing a self review
 ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
 🤏 Splitting feature into small PRs
 ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
 🧪 Providing/Improving test coverage
 🔐 Removing all sensitive information from the code and PR description
 🚩 Using a feature flag to protect the release
 👣 Providing comprehensive reproduction steps
 📑 Providing or updating our documentation
 🕛 Scheduling a pair reviewing session
 📱 Providing mobile support
 ♿ Providing accessibility support
Commit message and pull request title format standards
Note: Remove this section before opening the pull request
Make sure your PR title and commit message on squash and merge are as shown below

<commit type>: [JIRA-ticket-number] - <description>

Commit Types:

feat: New feature for the user (not a part of the code, or ci, ...).
fix: Bugfix for the user (not a fix to build something, ...).
change: Modifying an existing visual UI instance. Such as a component or a feature.
refactor: Restructuring existing code without changing its external behavior or visual UI. Typically to improve readability, maintainability, and performance.
test: New tests or changes to existing tests. Does not change the production code.
upcoming: A new feature that is in progress, not visible to users yet, and usually behind a feature flag.
Example: feat: [M3-1234] - Allow user to view their login history
